### PR TITLE
Search for reviews by Star Rating

### PR DESCRIPTION
**Pull request by vlbee**
Rebased with @Reettaphant search by date PR #506 
Adds ability to search by star-rating to help surface evergreen content. 

image

### CODE REVIEW

File: client-v2/src/components/FrontsCAPIInterface/SearchInput.tsx [hunk 1]
```diff
@@ -119,9 +126,25 @@ class FrontsCAPISearchInput extends React.Component<
     });
   };
 
+  public handleDropdownInput = (item: any, type: FilterTypes) => {
+    let newTags = [] as string[];
```
Comment (Reettaphant) [inline client-v2/src/components/FrontsCAPIInterface/SearchInput.tsx]: should these be called tags or something else?

File: client-v2/src/components/FrontsCAPIInterface/SearchInput.tsx [hunk 2]
```diff
@@ -169,12 +192,20 @@ class FrontsCAPISearchInput extends React.Component<
...
       q,
-      selected: { tags, sections }
+      selected: { tags, sections, desks, ratings }
     } = this.state;
 
-    const searchTermsExist = !!tags.length || !!sections.length || !!q;
+    const searchTermsExist =
+      !!tags.length || !!sections.length || !!desks.length || !!q;
+
+    const allTags = tags.concat(sections, desks, ratings);
```
Comment (Reettaphant) [inline client-v2/src/components/FrontsCAPIInterface/SearchInput.tsx]: should the ratings be concatenated with the tags or should they be stored on the state separately?

File: client-v2/src/services/capiQuery.ts [hunk 3]
```diff
@@ -64,14 +64,19 @@ const capiQuery = (
       })}`
     );
 
+    return response.json();
+  },
+  desks: async (params: any): Promise<CAPITagQueryReponse> => {
```
Comment (Reettaphant) [inline client-v2/src/services/capiQuery.ts]: why did this appear here in this pr?

File: client-v2/src/components/FrontsCAPIInterface/SearchInput.tsx (hunk 2 above)
Comment (vlbee) [reply client-v2/src/components/FrontsCAPIInterface/SearchInput.tsx]: This is need for the Commission Desks dropdown query

File: client-v2/src/components/FrontsCAPIInterface/FieldFilter.tsx [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+  <Downshift
+    itemToString={item => (item ? item.id : '')}
+    onChange={value => {
+      return onChange(value, filterType);
+    }}
+  >
+    {({
+      getItemProps,
+      getLabelProps,
+      getMenuProps,
```
Comment (Reettaphant) [inline client-v2/src/components/FrontsCAPIInterface/FieldFilter.tsx]: Where is this coming from? Is it defined anywhere?

**Pull request merged**